tests/test_cleaneval.py

```diff
@@ -0,0 +1,486 @@
+    tags_gold = tag_tp + tag_fn if tag_tp + tag_fn > 0 else 1
+    tag_precision = float(tag_tp) / tags_text
+    tag_recall = float(tag_tp) / tags_gold
+    precision_plus_recall = tag_precision + tag_recall if tag_precision + tag_recall > 0 else 1
+    tag_f_score = 2 * tag_precision * tag_recall / precision_plus_recall
+                                                                                                                                                        
+    return (100 * f_score, 100 * precision, 100 * recall, 100 * tag_f_score, 100 * tag_precision, 100 * tag_recall, tp, fp, fn, tag_tp, tag_fp, tag_fn)
+
+def write_alignment(diff, filename):
+	fh = file(filename, "w")
+	for tag, text_seg, gold_seg in diff:
+		if tag == "delete":
+			print("<" * 40, "(false positive)", file=fh)
+			print(" ".join(text_seg), file=fh)
+		if tag == "insert":
+			print(">" * 40, "(false negative)", file=fh)
+			print(" ".join(gold_seg), file=fh)
+		if tag == "equal":
+			print("=" * 40, file=fh)
+			print(" ".join(gold_seg), file=fh)
+	fh.close()
+
+def do_cleaneval(gold_html, cleaned_html):
+    '''
+    Arguments:
+        gold_html       String  the gold standard html
+        cleaned_html    String  the cleaned html returned from a library
+    Return:
+        eval_list       tuple   representing score on cleaned text evaluation comparing with the gold standard
+
+    >>> do_cleaneval(' <h> header <p> content', '<h> header <p> content') 
+    (100.0, 100.0, 100.0, 100.0, 100.0, 100.0, 5, 0, 0, 2, 0, 0)
+    '''
+    cleaned_html = normalize(cleaned_html)
+    text_words = re_WS.split(cleaned_html)
+    gold_words = re_WS.split(gold_html)
+    alignment = SequenceMatcher(None, text_words, gold_words)
+    diff = make_diff(alignment, text_words, gold_words)
+    eval_list = evaluate(diff)
+    return eval_list
+
+def do_jaccard(gold_html, cleaned_html):
+    cleaned_html = normalize(cleaned_html)
+    text_words = set(re_WS.split(cleaned_html))
+    gold_words = set(re_WS.split(gold_html))
+
+    intersection = gold_words.intersection(text_words)
+    union = gold_words.union(text_words)
+    
+    tp = len(intersection) / len(union)
+    fp = len(text_words.difference(gold_words)) / len(union)
+    fn = len(gold_words.difference(text_words)) / len(union)
+    result = __evaluate(tp, fp, fn, 0, 0, 0)
+    return result
+
+def do_jaccard_multiset(gold_html, cleaned_html):
+    cleaned_html = normalize(cleaned_html)
+    text_words = Multiset(re_WS.split(cleaned_html))
+    gold_words = Multiset(re_WS.split(gold_html))
+    intersection = gold_words.intersection(text_words)
+    union = gold_words.union(text_words)
+    
+    tp = len(intersection) / len(union)
+    fp = len(text_words.difference(gold_words)) / len(union)
+    fn = len(gold_words.difference(text_words)) / len(union)
+    result = __evaluate(tp, fp, fn, 0, 0, 0)
+    return result                                               
+
+def clean_cleaneval(raw_html):
```

Comment:
"cleaneval" is not a method, it is a dataset

this should be called `clean_none` since it is not doing any cleaning